website/routes.py

```diff
@@ -1280,20 +1319,18 @@ def make_url_map(app):
             '/project/<pid>/node/<nid>/beforeregister',
         ], 'get', project_views.register.project_before_register, json_renderer),
         Rule([
+            '/project/<pid>/draft/<draft_id>/register/',
+            '/project/<pid>/node/<nid>/draft/<draft_id>/register/',
+        ], 'post', project_views.drafts.register_draft_registration, json_renderer),
+        Rule([
```

Comment:
As far as I can tell all of these URLs are APIv1 endpoints; is that correct? If there are any user-facing endpoints,I believe that BrianG or Jeff are responsible for approving new URLs.